phpstan.neon.dist

```diff
@@ -4,22 +4,18 @@ includes:
 parameters:
 
     paths:
-        - config
-        - src
-        - tests
+        - config/
+        - src/
+        - tests/
 
     # The level 9 is the highest level
     level: 5
 
-    ignoreErrors:
-        - '#Parameter \#1 \$callback of method Illuminate\\Database\\Eloquent\\Factories\\Factory<Illuminate\\Database\\Eloquent\\Model>::afterCreating\(\)#'
-        - '#Access to an undefined property Tests\\Models\\User::\$active#'
-        - '#Access to an undefined property Tests\\Models\\UserCategory::\$position#'
-        - '#Access to an undefined property Tests\\Models\\Company::\$position#'
-        - '#Call to an undefined method Faker\\Generator::catchphrase\(\)#'
-        - '#Access to an undefined property Illuminate\\Database\\Eloquent\\Model::\$laravel_table_unique_identifier#'
+    databaseMigrationsPath:
+        - vendor/orchestra/testbench-core/laravel/migrations/
+        - tests/database/migrations/
 
-#    excludePaths:
```

Comment:
@szepeviktor I would to keep these lines commented here, could you please add them again at the end of the config?

Reply:
Done.